qa/suites/rgw/hadoop-s3a/s3a-hadoop-v28.yaml

```diff
@@ -9,15 +9,18 @@ overrides:
             osd default pool size: 2
             osd pool default pg num: 128
             osd pool default pgp num: 128
-            debug rgw: 20 
+            debug rgw: 20
             debug ms: 1
         ceph_test: true
-        ceph_dev: true
         ceph_dev_key: https://download.ceph.com/keys/autobuild.asc
         ceph_origin: upstream
```

Comment:
You should remove any instance of ``ceph_origin: upstream`` as it's not a valid option for ``ceph_origin`` and you've already got it defined as ``repository``.

Reply:
@andrewschoen thanks will remove both of them (dev_key and _origin).